modules/plugins/utility/new-file-template/new-file-template.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  ...
+}: let
+  inherit (lib.options) mkOption;
+  inherit (lib.types) attrsOf bool listOf str;
+  inherit (lib.nvim.types) mkPluginSetupOption;
+in {
+  options.vim.utility.new-file-template = {
+    enable = mkOption {
+      type = bool;
+      default = false;
+      description = "new-file-template.nvim: Automatically insert a template on new files in neovim";
```

Comment:
I'd prefer an option to add templates but if we are not doing that, lets either
a. tell people in the enable option how to add templates
b. add a documentation page for the plugin and link it here

Reply:
check the newest version are these docs ok